xdcr/rosmar_xdcr.go

```diff
@@ -186,30 +193,55 @@ func (r *rosmarManager) Stop(_ context.Context) error {
 	return nil
 }
 
-// opWithMeta writes a document to the target datastore given a type of Deletion or Mutation event with a specific cas.
-func opWithMeta(ctx context.Context, collection *rosmar.Collection, originalCas uint64, event sgbucket.FeedEvent) error {
-	var xattrs []byte
+// opWithMeta writes a document to the target datastore given a type of Deletion or Mutation event with a specific cas. The originalXattrs will contain only the _vv and _mou xattr.
```

Comment:
XDCR will be preserving the _sync xattr on the target, so I had anticipated originalXattrs would include that for setWithMeta to work as expected?  Or is that pending work for a later ticket.

Reply:
https://jira.issues.couchbase.com/browse/CBG-4281 for doing this, even though I think it's pretty trivial.